### ISSUE TITLE
Add Storybook page with general design principles

### ISSUE THREAD

**Issue by mattradikant** [Infineon/infineon-design-system-stencil#223]
Radikant wil provide input

**Comment by tishoyanchev** [comment]
NOTE:

It is necessary that we show what styles we are currently using, and explain to users how can they see the existing tokens in more detail.

For example, the font-family that we are using must be described somewhere so users can know what font family to use for their apps.

**Comment by mattradikant** [comment]
Yes, we would a general “Foundations” chapter in Storybook which contains the basic design styles separated in single pages for:

  *   Colors
  *   Typography
  *   Border radius
  *   Logo
  *   Interaction States


Von: Tisho Yanchev ***@***.***>
Datum: Montag, 16. Oktober 2023 um 10:04
An: Infineon/infineon-design-system-stencil ***@***.***>
Cc: Matthäus Eckert ***@***.***>, Author ***@***.***>
Betreff: Re: [Infineon/infineon-design-system-stencil] Add Storybook page with general design principles (Issue #223)

NOTE:

It is necessary that we show what styles we are currently using, and explain to users how can they see the existing tokens in more detail.

For example, the font-family that we are using must be described somewhere so users can know what font family to use for their apps.
